sklearn/tree/tests/test_tree.py

```diff
@@ -2466,7 +2503,7 @@ def test_missing_values_best_splitter_to_right(criterion):
 
 
 @pytest.mark.parametrize("criterion", ["entropy", "gini"])
-def test_missing_values_missing_both_classes_has_nan(criterion):
+def test_missing_values_best_splitter_missing_both_classes_has_nan(criterion):
```

Comment:
Is it possible to have a similar test for the random splitter?

Reply:
I added a new unit test here: [ff39dba](https://github.com/scikit-learn/scikit-learn/pull/27966/commits/ff39dba30ad26f38d9db2189dfb97c904053c866)